### PR TITLE
[4055] removed scrollbar styling attributes

### LINKED ISSUE #4055: Include a vertical scroll bar for dropdowns in desktop mode
**Description**:
Let's include a vertical scroll bar for dropdowns for the case when there are more than 4 options.

**Actual behavior**:
(Please make it so that the 5th item isn't partially showing)

image

**Expected behavior**:
If there are more than 4 options for the person to choose, the dropdown should have a vertical scroll like the one below:

image

**To Reproduce**:
1. Open a product with many dropdown options: https://master.d25kd40decjpue.amplifyapp.com/default/bundle-jeans-with-dynamic.html
2. Click on the dropdown.

**Additional context**:
Here you can see how the dropdowns behave: https://semantic-ui.com/modules/dropdown.html
If the dropdown is closer to the end of the screen, it should open going up:

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes #4055 

**Problem:**
* Small part of background visible from the left side of scroll

**In this PR:**
* Removed scrollbar styling

**Related PRs:**
* #4079

### CODE REVIEW

**Comment by pablobentivengo** [comment]
**Not ready**

**Comment by aleksandrakorolova** [comment]
Can reproduce with the bottom side of scrollbar
Screenshot 2022-02-14 at 10 16 36

**Comment by aleksandrakorolova** [comment]
Decided to create separate task for this issue

File: packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss [hunk 1]
```diff
@@ -63,29 +63,15 @@ $select-arrow-width: 14px !default;
...
-        &::-webkit-scrollbar {
-            /* stylelint-disable-next-line declaration-no-important */
-            display: auto !important;
-        }
-
-        &::-webkit-scrollbar-thumb {
-            /* stylelint-disable-next-line declaration-no-important */
-            display: auto !important;
-        }
+
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss:74]: not required line

File: packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss [hunk 2]
```diff
@@ -63,29 +63,15 @@ $select-arrow-width: 14px !default;
...
         overflow-y: auto;
         border-style: solid;
-        border-width: 0 1px 1px;
+        border-width: 0 0 1px 1px;
         border-color: transparent;
         transition: 200ms max-height;
         will-change: max-height;
         max-height: 0;
         -webkit-overflow-scrolling: touch;
         min-width: 100%;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss:73]: Same

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**